ministryofjustice/laa-apply-for-criminal-legal-aid
# CRIMAP-173 Use sub-navigation in dashboard

## Description of change

Adds subnavigation to the applications page

## Link to relevant ticket

https://dsdmoj.atlassian.net/browse/CRIMAP-173

## Screenshots of changes (if applicable)

![0 0 0 0_3000_completed_applications](https://user-images.githubusercontent.com/367349/199271506-cba8ed71-4b0b-4b01-ae68-b4593882fda9.png)

## How to manually test the feature

Complete and submit an application and then you should see it under submitted in 'you applications'. When you click the link for the application it should take you to the show page for that application. In developer tools, there will be an option to mark the application as 'returned'
